src/llmtuner/train/sft/trainer.py

```diff
@@ -28,6 +29,9 @@ class CustomSeq2SeqTrainer(Seq2SeqTrainer):
     def __init__(self, finetuning_args: "FinetuningArguments", **kwargs) -> None:
         super().__init__(**kwargs)
         self.finetuning_args = finetuning_args
+        if version.parse(torch.__version__) >= version.parse("1.13"):
```

Comment:
we should check if the badam is enabled here. btw, the torch version is always larger than 1.13 (see requirements)

Reply:
Thanks for your suggestion, the code has been adjusted.